docs/releasing.md

```diff
@@ -0,0 +1,69 @@
+# Overview
+
+This document describes the release flow and how to create a bundle release. 
+
+## Latest tag 
+
+Every change in the master branch which passes all checks triggers creating a `latest` tag and release.
+
+## Versioned releases
+
+Every tag pushed to the repository triggers new release. If the tag name matches the version pattern `x.y.0` (revision is `0`) Travis CI build creates a release tag `release-x.y`.
+
+## Create a release from the master branch
+
+Follow these steps to create a new bundle release 0.6.0:
+
+1. Checkout the master branch with the latest changes:
+
+    ```
+    git checkout master
+    git pull
+    ```
+
+2. Create a tag with proper release version:
+
+    ```
+    git tag 0.6.0
+    ```   
+
+3. Push the tag:
+
+    ```
+    git push {remote} 0.6.0 
+    ```
+    
+    for example push the tag to the upstream:
+    
+    ```
+    git push upstream 0.6.0 
+    ```
+    
+
+4. After the Travis CI jobs finish, go to the **releases** tab and the **branches**. You can find there new release and new release branch.
```

Comment:
After ~the~ Travis CI jobs finish, go to the `**Releases**` tab and the `**Branches**`. `There you can find a new release and a new release branch.`